MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -118,11 +118,16 @@ open class RouteProgress: NSObject {
      */
     public var currentLegProgress: RouteLegProgress!
     
+    public typealias TimedCongestionLevel = (CongestionLevel, TimeInterval)
```

Comment:
Since this typealias is public and the method arguments that use it are also public, the typealias needs documentation.